File: TSLMS_Backend/TimesheetService/Models/Project.cs
namespace TimesheetService.Models
{
    public class Project
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Code { get; set; } = string.Empty;
        public string? Description { get; set; }
        public bool IsActive { get; set; } = true;
        public DateTime CreatedAtUtc { get; set; }
        public DateTime UpdatedAtUtc { get; set; }
    }
}

File: timesheet-leave-management-frontend/src/app/models/project.ts
export interface Project {
  id: string;
  name: string;
  code: string;
  description: string | null;
  isActive: boolean;
  updatedAtUtc: string;
}

export interface CreateProjectDto {
  name: string;
  code: string;
  description?: string | null;
  isActive: boolean;
}

export interface UpdateProjectDto extends CreateProjectDto {}
